contrib/docker/docker-entrypoint.sh

```diff
@@ -4,14 +4,18 @@ set -ex
 if [ "$#" -ne 0 ]; then
     exec "$@"
 elif [ "$SUPERSET_ENV" = "development" ]; then
-    superset worker &
+    celery worker --app=superset.sql_lab:celery_app --pool=gevent -Ofair &
     # needed by superset runserver
     (cd superset/assets/ && yarn && yarn run sync-backend)
     (cd superset/assets/ && yarn run dev) &
-    flask run -p 8088 --with-threads --reload --debugger --host=0.0.0.0
+    FLASK_APP=superset:app flask run -p ${SUPERSET_PORT} --with-threads --reload --debugger --host=0.0.0.0
```

Comment:
@mistercrunch can you validate it is the correct command? I have no idea :)

Reply:
command looks good